features/apps/finder_frontend.feature

```diff
@@ -19,12 +19,6 @@ Feature: Finder Frontend
     When I search for "<keywords>"
     Then I should see some search results
     And the search results should be unique
-    And search analytics for "<keywords>" are reported
-    When I go to the next page
-    Then the "contentsClicked" event is reported
-    When I click result 1
-    Then the "navFinderLinkClicked" event for result Search.1 is reported
-    And the "UX" event for result click is reported
```

Comment:
Bit of a 'meta' question sorry - does the fact that these tests actually create analytics data mean we should never have/use/write tests like this? Surely, we're always polluting data and unless that can be subsequently ignored it'll remain an issue. 

Could we add/change something into the analytics data being created here - 'TEST_PLEASE_IGNORE' for example - so that it can be reliably be ignored.

As a general thought, I wonder if this is the best way to test that analytics data is being fired and captured 🤔 

Do we know if it's specific keywords that cause the pollution or just that fact that anything in there is an issue? Just thinking if we can exclude tests based on specific keywords.

Sorry, a lot of questions...

Reply:
It turns out the clickboost score is being distorted. The LTR model in the old search-api has added logic to mitigate this, but the new search is just using the clickboost scores as is, because why wouldn't you? 😱 

The test data just made it easier to spot. One of the things that the PA stumbled on was that the order of results for "passport" looked odd.

I do wonder whether checking that the analytics are being reported need to be a smokey test at all, or yes, should have someway of identifying itself as test data that should be ignored.